scripts/security/certs-create.sh

```diff
@@ -9,7 +9,7 @@
 rm -f *.crt *.csr *_creds *.jks *.srl *.key *.pem *.der *.p12
 
 # Generate CA key
-openssl req -new -x509 -keyout snakeoil-ca-1.key -out snakeoil-ca-1.crt -days 365 -subj '/CN=ca1.test.confluent.io/OU=TEST/O=CONFLUENT/L=PaloAlto/S=Ca/C=US' -passin pass:confluent -passout pass:confluent
+openssl req -new -x509 -keyout snakeoil-ca-1.key -out snakeoil-ca-1.crt -days 365 -subj '/CN=ca1.test.confluent.io/OU=TEST/O=CONFLUENT/L=PaloAlto/ST=Ca/C=US' -passin pass:confluent -passout pass:confluent
```

Comment:
Do we understand when to use `S` or when to use `ST`?  What's the better practice?

Reply:
Not really.  The documentation is weak.  `openssl req` produces this error warning if you use `S`: 

```
Subject Attribute S has no known NID, skipped
```

Change the attribute to `ST` and it goes away.  I assume that X509 has a different set of valid attributes or keytool / pkcs12 doesn't validate it.